sentences/pl/light_HassLightSet.yaml

```diff
@@ -12,3 +12,5 @@ intents:
       # color
       - sentences:
           - "(Ustaw | Zmień) [kolor] <name> na {color}"
+      - sentences:
+          - "(Ustaw | Zmień) [wszystkie] [kolory] <name> na {color}"
```

Comment:
You can just allow multiple different phrases in one `Optional word` syntax like `[kolor|kolory]` no need for additional sentence.
If in doubt you can check https://voice.home-assistant.io/docs/intents/matching-syntax/#sentence-templates-syntax for formatting tips.

Reply:
And while we are here you've added a new prefix `Przestaw` in other places 😉 

At this point this should probably be added to the https://github.com/home-assistant/intents/blob/main/sentences/pl/_common.yaml as a expansion_rule to not repeat ourselves all over the place.

Reply:
One problem which I see is that in polish language we not say: "Zmień wszystkie kolor lamp na czerwone." The correct one is "Zmień wszystkie kolory lamp na czerwone." That was the reason why I created new sentence 😀 We should some how cover this differences

Reply:
The issue here is even more with the entire sentence word order not even this particular word (which you could solve with nesting the optional phrase like so `(Ustaw | Zmień) [kolor | [wszystkie] kolory] <name> na {color}`).
But even then you end up with a sentence like in your example `Zmień wszystkie kolory lamp na czerwone.` and AFAIK it won't work like that because it lacks the device name and you are trying to use something like device_class here. Did you manage to test it? Could you show an example of proper use please?

Reply:
Yep, I will check it and prepare correct sentence

Reply:
OK, you have a right, I forgot add area to this sentence. Now I updated tests and added more examples. 

PS. Looking through the tests, I noticed wrong device names for the tests, I also corrected them 😀

Reply:
BTW. Regarding prefixes I reorganised `expansion_rules` and I update tests 😉 I didn't want to go too crazy because I don't know how deep we want to go into this. Now it's worth adding a bit more tests

Reply:
Great, now it looks amazingly good, will check everything today's evening but at the first glance it's all good. Thank you for your contribution Witold 👍